### PR TITLE
Fix: add missing docstrings for exported symbols.

### PR DESCRIPTION
**Pull request by klihub**
Add a few missing docstrings for exported types, functions in pkg/cgroups and pkg/metrics.

### CODE REVIEW

File: pkg/metrics/metrics.go [hunk 1]
```diff
@@ -12,8 +12,10 @@ var (
 	log                  = logger.NewLogger("metrics")
 )
 
+// InitCollector is the type for functions that initialize collectors.
 type InitCollector func() (prometheus.Collector, error)
 
+// RegisterCollector registers the names prometheus.Collector for metrics collection.
```
Comment (marquiz) [inline pkg/metrics/metrics.go]: nit: maybe it's just me but I don't understand the meaning of *the names prometheus.Collector* 😄

File: pkg/cgroups/cgroupid.go [hunk 2]
```diff
@@ -10,12 +10,14 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// CgroupID is a cgroup id to path map/cache.
```
Comment (marquiz) [inline pkg/cgroups/cgroupid.go]: I don't fully understand this either 😉

File: pkg/metrics/metrics.go (hunk 1 above)
Comment (klihub) [reply pkg/metrics/metrics.go]: Argh... should have been *named*. Fixed.

File: pkg/cgroups/cgroupid.go (hunk 2 above)
Comment (klihub) [reply pkg/cgroups/cgroupid.go]: Hmm... fair enough. Let's try to clarify it then.

**Pull request merged**